### DISCUSSION TITLE
Could you add the logo of Scroll please?

### DISCUSSION THREAD

**Discussion by dabaicai66666** [rainbow-me/rainbowkit#1637] [Feedback]
Hi, 
Our project integrates rainbow to wallet login, but the chain logo is not displayed when selecting the scroll chain. Could you please support it? Thanks!

**Comment by 0xmkh** [comment]
@dabaicai66666 Have you tried to adding custom icon and custom background color ? Feel free to checkout the docs [here](https://www.rainbowkit.com/docs/custom-chains)
